wdl/call_variants.wdl

```diff
@@ -137,6 +194,193 @@ workflow CallVariants {
+            --tumor_sample_name '~{tumor_sample_name}' \
+            -I:tumor '~{tumor_bam}' \
+            ~{"--normal_sample_name '" + normal_sample_name + "'"} \
+            ~{"-I:normal '" + normal_bam + "'"} \
+            --normal_panel '~{panel_of_normals}' \
+            --dbsnp '~{germline_resource}' \
+            --downsample_to_coverage ~{downsample_to_coverage} \
+            --max_alt_alleles_in_normal_count ~{max_alt_alleles_in_normal_count} \
+            --max_alt_alleles_in_normal_qscore_sum ~{max_alt_alleles_in_normal_qscore_sum} \
+            --fraction_contamination $fraction_contamination \
+            --tumor_f_pretest ~{tumor_f_pretest} \
+            --initial_tumor_lod ~{initial_tumor_lod} \
+            --tumor_lod ~{tumor_lod} \
+            --out '~{call_stats}' \
+            --coverage_file '~{coverage_wig}' \
+            --power_file '~{power_wig}' \
+            --vcf '~{mutect1_vcf}'
+    >>>
+     
+    output {
+        File mutect1_call_stats = call_stats      
+        File mutect1_coverage_wig = coverage_wig
+        File mutect1_power_wig = power_wig
+        File mutect1_vcf = mutect1_vcf
+        File mutect1_vcf_idx = mutect1_vcf_idx
+    }
+
+    runtime {
+        docker: runtime_params.docker
+        bootDiskSizeGb: runtime_params.boot_disk_size
+        memory: runtime_params.machine_mem + " MB"
+        runtime_minutes: runtime_params.runtime_minutes
+        disks: "local-disk " + runtime_params.disk + " HDD"
+        preemptible: runtime_params.preemptible
+        maxRetries: runtime_params.max_retries
+        cpu: runtime_params.cpu
+    }
+ }
+ 
+task MergeMutect1ForceCallVCFs {
+    input {
+        File interval_list
+        File ref_fasta
+        File ref_fasta_index
+        File ref_dict
+
+        Array[File?] mutect1_vcfs # not gzipped
+        Array[File?] mutect1_vcfs_idx # not gzipped
+        File? force_call_alleles # gzipped
+        File? force_call_alleles_idx # gzipped
+
+        Runtime runtime_params
+    }
+
+    String base = "merged_mutect1"
+    String mutect1_vcf = base + ".vcf.gz"
+    String mutect1_pass_no_genotypes_vcf = base + ".pass.no_genotypes.vcf.gz"
+    String mutect1_pass_no_genotypes_forcecalled_vcf = base + ".pass.no_genotypes.forcecalled.vcf.gz"
+    String mutect1_pass_no_genotypes_forcecalled_dedup_vcf = base + ".pass.no_genotypes.forcecalled.dedup.vcf.gz"
+    String mutect1_pass_no_genotypes_forcecalled_dedup_vcf_idx = mutect1_pass_no_genotypes_forcecalled_dedup_vcf + ".tbi"
+    String mutect1_pass_no_genotypes_forcecalled_dedup_subset_vcf = base + ".pass.no_genotypes.forcecalled.dedup.subset.vcf.gz"
+    String mutect1_pass_no_genotypes_forcecalled_dedup_subset_vcf_idx = mutect1_pass_no_genotypes_forcecalled_dedup_subset_vcf + ".tbi"
+
+    command <<<
+        set -euxo pipefail
+
+        # Concat all samples VCFs from shardX and compress the merged output
+        bcftools concat ~{sep=" " mutect1_vcfs} -Oz > '~{mutect1_vcf}'
```

Comment:
~{sep=" " mutect1_vcfs} also won't work if that sample name has a whitespace. 
`~{sep="' " prefix(" '", mutect1_vcfs)}'` should do